onyo/lib/onyo.py

```diff
@@ -38,57 +38,129 @@ def __init__(self, path: Union[Path, str], init: bool = False) -> None:
 
     @property
     def assets(self) -> set[Path]:  # pyre-ignore[11]
+        """
+        A `set` containing the `Path`s relative to repo.root to all assets of an
```

Comment:
Nit: I would write this a bit differently. Also use backticks for `Repo.root`, and capitalize the `R` in `Repo.root`

> A `set` containing all asset `Path`s relative to `Repo.root`.